rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/LoggersWindow.java

```diff
@@ -90,7 +92,18 @@ private void build() {
         loggersGrid.setShowAllRecords(true);
 
         ListGridField loggerField = new ListGridField("logger", "Logger", 385);
-        loggerField.setRequired(true);
+        // prevent logger field validation if cell is empty or contains white spaces
+        Validator loggerValidator = new CustomValidator() {
+            @Override
+            protected boolean condition(Object value) {
+                if (value == null || value.toString().trim().isEmpty()) {
+                    return false;
+                }
+                return true;
```

Comment:
You could have a single `return value != null && !value.toString().trim().isEmpty()`

Reply:
@AlexFalessi Nice, done !